### ISSUE TITLE
Mobile Interface: Canvassing Record Input Interface

### ISSUE THREAD

**Issue by aarontitus** [aarontitus/crisiscleanup-legacy#117]
_Original author: v...@aarontitus.net (January 01, 2013 23:21:38)_

The Mobile Canvassing Record Input Interface should be similar to the Assessment/Intake/Work Order Input Interface, but much simpler. See #114 and #116. The interface should include:
- Address (Required). This should work exactly as the Assessment/Intake/Work Order Input Interface, except: If the lat/long is provided by the mobile device, then the Google API should guess all of the Address fields.  Note: For the mobile device interface, the street number should be placed in a separate field for easy manual correction, as that is the most likely portion of the address to be incorrect.
- Lat/Long (Required). This should work exactly as the Assessment/Intake/Work Order Input Interface, Except: The Lat/Long should be requested from the (mobile) device first. If the mobile device gives a Lat/Long, then the address is guessed.  If the mobile device does not give a lat/long, then an address is manually required, and the Google API fills in the Lat/Long.
- Status (Required). Include the following Buttons or check-boxes (E.g. &quot;No Needs: Helped by Others&quot;), or nested options (e.g. &quot;No Needs&quot; -&gt; &quot;Helped by Others&quot;). Valid Statuses (or their aliases):
  - No Needs: No Help Wanted
  - No Needs: Unaffected
  - No Needs: Helped by Others
  - No Needs: Rejected (Please Explain)
  - No Needs: Needs are Out of Scope
  - Follow Up: Nobody Home
  - Follow Up: Owner Not Present
  - Follow Up: May Need Help in the Future
  - Follow Up: Needs Help, but Not Ready
  - Needs Help/ Assessed [If this selection is chosen in the interface, then the user is forwarded directly to the Assessment/ Intake form, before creating a Canvassing Record. After the assessment is complete, a Work Order record is created, and also Canvassing Record at the same time]
- Status Notes (Optional). A text field with additional explanation of the status, if desired.
- Name (Optional). A text field
- Phone Number (Optional)

Related: #116, Desktop Interface
Note: #114 must be completed first.

_Original issue: http://code.google.com/p/sandy-disaster-recovery/issues/detail?id=117_
